requirements.txt

```diff
@@ -1,5 +1,5 @@
 codecarbon==2.2.3
-pandas==2.0.2
+pandas==2.0.3
```

Comment:
@Dheemant-Dixit Is pandas upgrade required?

Reply:
Yes, there was a conflict when 2.0.2 was used.